docs/specification.md

```diff
@@ -105,6 +108,15 @@ type config =
 
   ; voting_period : voting_period
   // ^ Determine the voting_period length.
+
+  ; proposal_flush_time : seconds
+  // ^ Determine how long after the proposal is proposed to be able to be `flushed`.
```

Comment:
Probably needs a bit rephrasing for clarity. something like "minimum time that should pass after a proposal's submission before it can be flushed"..

Reply:
Updated